BoxContentSDK/BoxContentSDK/Operations/BOXAPIOperation.m

```diff
@@ -527,8 +539,12 @@ - (void)processResponse:(NSURLResponse *)response
 
 - (void)finishURLSessionTaskWithData:(NSData *)data response:(NSURLResponse *)response error:(NSError *)error
 {
-    [self processResponse:response];
-    [self processResponseData:data];
+    if (response != nil) {
+        [self processResponse:response];
```

Comment:
I don't recall the exact reason, given that we might have a valid intermediate HTTPResponse before this finish call, we might have wanted to keep the previous response if the last response is nil. But we can always bring it back if needed. Removed.